guides/common/modules/proc_cloning-ansible-roles-from-vcs.adoc

```diff
@@ -0,0 +1,24 @@
+[id="cloning-ansible-roles-from-vcs-{context}"]
+= Cloning Ansible roles from VCS
+
+You can clone Ansible roles to your {SmartProxy} from the {ProjectWebUI}.
+
+.Prerequisites
+* A {SmartProxy} with VSC integration for Ansible.
+For more information, see xref:setting-up-vsc-integration-for-ansible_{context}[].
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Configure* > *Ansible* > *Roles*
+. Click on `Clone from VCS...` in the toolbar.
+Follow the wizard to clone an Ansible role from VCS.
+
+Note: Currently, only links using *HTTPS* are supported
+
+* Click *Examine* to select a git revision from the list of branches and tags.
+* Select *Skipping existing* to overwrite existing Ansible roles on your {SmartProxy}.
```

Comment:
...to _not_ overwrite...?

Reply:
I rephrased this section to make it a bit clearer. The UI also has a HelperText that explains this.